pkgs/development/python-modules/soundcard/default.nix

```
@@ -0,0 +1,47 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchPypi,
+  lib,
+  libpulseaudio,
+  numpy,
+  setuptools,
+  testers,
+}:
+
+buildPythonPackage rec {
+  pname = "SoundCard";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
    pname = "SoundCard";
    inherit version;
```

since `pname` is no longer the same as the pypi name, we have to manually provide the name